TranslucentTB/taskbar/taskbarattributeworker.cpp

```diff
@@ -307,22 +307,54 @@ TaskbarAppearance TaskbarAttributeWorker::GetConfig(taskbar_iterator taskbar) co
 
 	if (m_Config.SearchOpenedAppearance.Enabled && m_CurrentSearchMonitor == taskbar->first)
 	{
-		return WithPreview(txmp::TaskbarState::SearchOpened, m_Config.SearchOpenedAppearance);
+		const std::optional<Rule> rule = m_Config.SearchOpenedAppearance.FindRule(window);
```

Comment:
Move this code to a dedicated function to avoid duplication.

Reply:
If there is any color picker opened, it applies the color it's currently selecting, to implement a live preview.